packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -337,7 +388,51 @@ class TextSelectionOverlay {
   /// {@macro flutter.widgets.SelectionOverlay.showToolbar}
   void showToolbar() {
```

Comment:
@camsim99 Does it look like you'll be able to tie your spell check menu in here somewhere?  I guess if users are customizing their context menu via contextMenuBuilder then they will expect to be able to update the spellcheck menu too.

Hopefully it should also be easy to build the actual UI for your menu using the toolbar widgets in this PR.